src/applications/travel-pay/constants.js

```diff
@@ -70,15 +70,15 @@ export const STATUSES = {
     description:
       'The Travel Clerk determined the claim does not qualify for a full reimbursement. Instead, they approved a partial payment and a Partial Payment letter was sent to you.',
     definition:
-      'Some of the expenses you submitted aren’t eligible for reimbursement. You can review the decision letter on the Claim Details page soon for more information.',
+      "Some of the expenses you submitted aren't eligible for reimbursement. You can review the decision letter for more information.",
```

Comment:
Why double quotes here? All the other appear to be single. Maybe a linting thing?